recipes/redradist-icc/all/conanfile.py

```diff
@@ -0,0 +1,108 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration, ConanException
+import os
+
+
+class ICCConan(ConanFile):
+    name = 'redradist-icc'
+    homepage = 'https://github.com/redradist/Inter-Component-Communication'
+    license = 'MIT'
+    url = 'https://github.com/conan-io/conan-center-index'
+    description = "I.C.C. - Inter Component Communication, This is a library created to simplify communication between " \
+                  "components inside of single application. It is thread safe and could be used for creating " \
+                  "components that works in different threads. "
+    topics = ("thread-safe", "active-object", "communication")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        'fPIC': [True, False],
+    }
+    default_options = {
+        'shared': False,
+        'fPIC': True,
+    }
+    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
```

Comment:
It's rare to need both find package, can you please try

```suggestion
    generators = "cmake", "cmake_find_package_multi"
```

Reply:
User can specify any of this generators ...
What will happen if user specify `cmake_find_package`, but my package supports only `cmake_find_package_multi` ?

Reply:
This is not related to the consumer... this is for the build... technically either should work but multi matches more with the "manual" workflow

Reply:
I've heard from @SpaceIm that both generators are needed:
- `cmake_find_package` generates things like `FindZLIB.cmake`
- `cmake_find_package_multi` generates things like `Boost-config.cmake`

the problem if you leave just `cmake_find_package_multi`, it won't generate files like `FindZLIB.cmake`.
and it's very unlikely the library has `find_package(ZLIB CONFIG)`, it will have just `find_package(ZLIB)`.
so, although conan generates `ZLIB-config.cmake`, library will never pick it, and by default will just use built-in `FindZLIB.cmake`. that's not the effect we expect.
so practically, every recipe always needs both `cmake_find_package` + `cmake_find_package_multi`.

Reply:
This library has no dependency, so cmake_find_package[_multi] generators are useless